identity/app/views/completeConsents.scala.html

```diff
@@ -0,0 +1,19 @@
+@(
+    idRequest: services.IdentityRequest,
+    idUrlBuilder: services.IdentityUrlBuilder,
+    returnUrl: String
+)(implicit request: RequestHeader)
+
+@import common.LinkTo
+
+<div class="identity-wrapper monocolumn-wrapper">
+    <section class="identity-forms-message">
+        <h1 class="identity-title">Thank you for letting us know your preferences</h1>
+        <div class="identity-forms-message__body">
+            <p>You can change your preferences anytime by visiting <a class="u-underline" href="@idUrlBuilder.buildUrl(controllers.editprofile.routes.EditProfileController.displayEmailPrefsForm(false, None).url, idRequest)">Email Preferences</a>.</p>
+        </div>
+        <aside class="identity-forms-message__body">
+            <a class="manage-account__button manage-account__button--center manage-account__button--main" href="@LinkTo(returnUrl)">Continue</a>
```

Comment:
Can you add an arrow to this button? otherwise this is looking great. Thank you very much!